src/modules/navigation/components/Navigation/NavigationSection/NavigationSection.tsx

```diff
@@ -21,6 +23,14 @@ const NavigationSection: FC<NavigationSectionProps> = ({
 }) => {
 	// Needed for overlay state. Dropdown state is saved in NavigationDropdown component
 	const [isHamburgerMenuOpen, setIsHamburgerMenuOpen] = useState(false);
+	const history = useSelector(selectHistory);
+
+	// Close dropdowns when the url path changed
+	useEffect(() => {
+		if (history?.[1] !== currentPath && isHamburgerMenuOpen) {
+			setIsHamburgerMenuOpen(false);
+		}
+	}, [history, currentPath, isHamburgerMenuOpen]);
```

Comment:
Can't you handle this closing in the onClick handler of a navigation item or maybe in the onClick handler of the wrapper around the navigation items.
Or doesn't the event bubble to the top anymore?